recipes/protobuf-c/all/conanfile.py

```diff
@@ -0,0 +1,84 @@
+import functools
+
+from conans import ConanFile, CMake, tools
+import os
+
+
+class ProtobufcConan(ConanFile):
+    name = "protobuf-c"
+    license = "https://github.com/protobuf-c/protobuf-c/blob/master/LICENSE"
```

Comment:
https://github.com/conan-io/conan-center-index/blob/master/docs/reviewing.md#license-attribute

Looks like a modified https://spdx.org/licenses/BSD-2-Clause

```suggestion
    license = "LicenseRef-LICENSE"
```